python-lib/lal/handlers/lal_handler.py

```diff
@@ -112,8 +113,7 @@ def label(self, data, user):
 
         lbl_id = self.create_label_id() if existing_meta_record.empty else existing_meta_record.iloc[0][self.lbl_id_col]
         raw_data = self.classifier.get_raw_item_by_id(data_id)
-
-        serialized_label = self.classifier.serialize_label(data.get('label')) if data.get('label') else None
+        serialized_label = self.classifier.format_for_saving(data.get('label')) if data.get('label') else None
```

Comment:
I don't think I understand the need for another method. The idea was that there's a label object in the backend that needs to be written and read from dataset. For this each classifier has `serialize_label` and `serialize_label` method pair.

Reply:
Indeed, included everything in serialize_label, thanks! Fixed in f05bf27